packages/shared/src/hooks/useBookmarkProvider.ts

```diff
@@ -1,25 +1,20 @@
-import { useRef } from 'react';
 import { useActiveFeedNameContext } from '../contexts';
 import { SharedFeedPage } from '../components/utilities';
+import { UseBookmarkProviderProps, useJustBookmarked } from './bookmark';
 
 interface UseBookmarkProviderReturn {
   highlightBookmarkedPost: boolean;
 }
 
-interface UseBookmarkProviderProps {
-  bookmarked?: boolean;
-}
-
 const useBookmarkProvider = ({
   bookmarked = false,
 }: UseBookmarkProviderProps): UseBookmarkProviderReturn => {
-  const isBookmarked = useRef<boolean>(bookmarked);
   const { feedName } = useActiveFeedNameContext();
-  const justBookmarked = bookmarked !== isBookmarked.current && bookmarked;
+  const { justBookmarked, wasBookmarked } = useJustBookmarked({ bookmarked });
```

Comment:
This still supports the old one (highlighting old bookmarks) vs now also showing the new popup I guess?

Reply:
Yes, should still support as intended. Just changed the variable name from `isBookmarked` to `wasBookmarked` - to be clearer.
![Screenshot 2024-07-11 at 9 46 23 PM](https://github.com/dailydotdev/apps/assets/13744167/7b98b08c-dce4-44e5-b00e-441fd357c23c)